packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -444,17 +443,17 @@ export const HelpCenterContactForm = () => {
 		);
 	};
 
-	const { isFetching: isFetchingUrls, data: links } = useJetpackSearchAIQuery(
+	const { isFetching: isFetchingGPTUrls, data: links } = useJetpackSearchAIQuery(
```

Comment:
Optional: `useJetpackSearchAIQuery` should accept `enabled` param. That would be more accurate than `shouldLoadGptAnswer ? debouncedMessage : ''`